gateways/node_api_gateway.py

```diff
@@ -56,7 +59,7 @@ def get_address_balance(self, address: str) -> Optional[dict]:
         return self.hathor_core_client.get(
             ADDRESS_BALANCE_ENDPOINT,
             params={"address": address},
-            timeout=10,
+            timeout=14,  # lambda timeout 15s
```

Comment:
You need to be careful because some Lambdas make more than one request to the outside. This is one of them.

See https://github.com/HathorNetwork/hathor-explorer-service/pull/93, which is the PR that set the timeout as 10 here.

Apparently it should remain as 10 if the Lambda timeout is 15.